src/Tasks/XamlRules/CSharp.BrowseObject.xaml

```diff
@@ -1,4 +1,3 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Hmm...wondering if we should also update the copyright header in these files.

Reply:
That's my next PR! 😏